website/docs/advanced/config-v2-migration-guide.md

```diff
@@ -6,27 +6,37 @@ description: A detailed guide on how to migrate from Config V1 to Config V2.
 
 This guide will help you migrate from Config V1 to Config V2.
 
+:::info
+This migration guide is for ConfigCat users who have Config V2 enabled. If you don't have Config V2 enabled, you can [contact us](https://configcat.com/support/) to add you to the Beta program.
+:::
+
 ## What is Config V2?
 
-Config V2 is the next generation of ConfigCat. It comes with a new dashboard, API, SDKs, and features. It's not backward compatible with Config V1. Read more about the new features in the [Config V2 Overview](advanced/config-v2).
+Config V2 is the next generation of ConfigCat. It comes with a new Dashboard, Public Management API and SDKs.
+
+Config V2 supports all the features of V1, so you can continue using those, but it offers interesting new features as well. However, you won't be able to use the new features with the V1 versions of the Dashboard, Public Management API and SDKs.
+
+Read more about the new features in the [Config V2 Overview](/advanced/config-v2).
 
 ## A few things to consider before migration
 
+- Migration from Config V1 means that a V2 config will be copied from the V1 config. The V1 config will remain unchanged and accessible. The V2 version will have a new SDK key for each environment.
 - Config V2 and V1 are completely separate. They don't affect each other in any way. You can use them side by side.
-- Every newly created config is a V2 config.
-- You can migrate your V1 configs to V2. That will mean that a V1 and a V2 version of your config will be available at the same time. The V2 version will have a new SDK key for each environment.
-- There is no automatic sync between the V1 and V2 configs. You have to manually update the V2 config if you make changes to the V1 config and vice versa.
-- Once started it's recommended to migrate your V1 configs to V2 as quickly as possible to avoid confusion.
-- There is no pressure to migrate. You can stay on V1 for a long time.
+- Every newly created config will be a V2 config by default.
+- There is no automatic sync between the V1 and V2 configs. If you want to keep them in sync, you have to manually update the V2 config when you make changes to the V1 config and vice versa.
+- There is no pressure to migrate. Although we have plans to phase out V1 eventually, you can stay on it for a long time.
+- Once the migration process is started, it's recommended to migrate your V1 configs to V2 as quickly as possible to avoid confusion. 
 
 ## Migrating from Config V1 to Config V2
 
+To migrate from Config V1 to Config V2, you have to copy your V1 config to a new V2 one and update your applications to use the new config. The migration process is the following:
```

Comment:
Ez nem igazán pontos megfogalmazás, még mindig áll, amit Lali írt: "A you have to create rész szerintem nem fedi így a valóságot, mert így én úgy tudom érteni, hogy csinálni kell egy új configot, de az egy másik flow."

```suggestion
The migration process starts with copying your V1 config to a new V2 one and updating your applications to use the new config. The complete process is the following:
```